lib/active_merchant/billing/gateways/netbanx.rb

```diff
@@ -331,41 +331,20 @@ def headers
       def error_code_from(response)
         unless success_from(response)
           case response['errorCode']
-          when '3002' then STANDARD_ERROR_CODE[:invalid_number] # You submitted an invalid card number or brand or combination of card number and brand with your request.
```

Comment:
I'm a little torn on removing this kind of inline documentation... I wonder if there's some similar structure rubocop doesn't mind... but again NBD, these messages should be available in the response anyway.